src/pages/signup-affiliates/components/wizard-component/wizard-footer.tsx

```diff
@@ -58,21 +58,23 @@ const WizardFooter = ({
 }: WizardComponentTypes) => {
     const buttonHandler = React.useCallback(
         (button_type: ButtonType): void => {
+            const prev_step = step - 1
```

Comment:
Why do you use different naming notation for variables,  eg. `const buttonHandler`, `const prev_step`?

Reply:
@dan-deriv as far as I know in deriv-com we're using snake_case for variables and camelCase for functions/hooks